### PR TITLE
Project Setup guide

### PR DESCRIPTION
**Pull request by hofmannedv**
### All Submissions:

Corrections of the project setup guide

### CODE REVIEW

File: chapters/project_guide.rst [hunk 1]
```diff
@@ -1,153 +1,161 @@
...
 
 Here you will see the suggested structure for a project structure in Jina.
 
 .. contents:: Table of Contents
     :depth: 2
 
 
-Install Jina
-====================
+Installing Jina
```
Comment (Yongxuanzhang) [inline chapters/project_guide.rst:18]: Why use installing?

File: chapters/project_guide.rst [hunk 2]
```diff
@@ -1,153 +1,161 @@
...
 Our Flows will need some :term:`Pods<Pod>`, so we also create a dedicated `pods` folder for them.
 
 App.py
----------
+------
 
 ``/app.py``
 
-And of course, we need our main app, we have this file living directly under the main directory.
+Finally, the main app needs a dedicated place.,This file will be living directly under the main directory.
```
Comment (Yongxuanzhang) [inline chapters/project_guide.rst]:
```suggestion
Finally, the main app needs a dedicated place. This file will be living directly under the main directory.
```

**Comment by Yongxuanzhang** [comment]
Hi, thank you so much for contributing to improving this section. This helps a lot! 👍

**Review by bwanglzu — APPROVED** [review]
LGTM if you can fix @Yongxuanzhang 's comments

File: chapters/project_guide.rst (hunk 2 above)
Comment (hofmannedv) [reply chapters/project_guide.rst]: Oh yes, the comma needs to be removed. I will fix that quickly.

**Review by FionnD — APPROVED** [review]

**Pull request merged**